### ISSUE TITLE
markdown: multi-line comments keep moving to the right

### ISSUE THREAD

**Issue by henri-hulski** [prettier/prettier#4257]
**Prettier 1.11.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBidACCAHOAnAQxgjwB0pyADagKwGdyBLAW2xJg2AwAsYZsAInAA2ceBgC+GAGZ4IzDAHIAAmHxwARoWEB6Xvx058REnUVNW7ThjowicSTLkLFavJu067AczPlycAAebHgcACZw0gQArsIcANrkGDx8giJicAAUtniMUN6UOozwzHQ6ACTAtvaUYNF47rAAkiVNYZQSlACUSRg6AFS9ycMjo2PjGAAKAEoA8pMAykicQxNr68nudGxQdHDLwAB0xxKrG+fDp1AXNxj9OuQAulTU5CAANCA4MIzQdMigAgNCAAd0mQIQ-xQBGEIIIAE9-p8CLZkFFhHtPloCGAANZiBbYHF5bzIGB4aJwT5hCBgNEwzEgPJ7UKTQjeZgEekYqkgeiBABChDxBIIzDgABk8nBuYyINF+AqAEyy3lEvAs5AgTl4XE0kFQD4gbC5WAAdUYYRg3GQAA4AAyfE0QPZmwjYLUmuAsgBuMs+7gAjtFGO42QQOVykOjGXtmIxVZ86CTRABFaIQeBkim8uwaC1Wm1IJWfckERjCEkAYXknK1UGg-pA0T2ABUCBooTG4BIJEA)
```sh
--arrow-parens always
--parser markdown
--no-semi
--single-quote
```

**Input:**
````markdown
### operator

```js
import { httpDelete } from '@cerebral/http/operators'
import { state } from 'cerebral/tags'

export default [
  httpDelete(string`/items/${state`currentItemId`}`)
  /*
                              PROPS: {
                                response: {...}
                              }
                            */
]
```

````

**Output:**
````markdown
### operator

```js
import { httpDelete } from '@cerebral/http/operators'
import { state } from 'cerebral/tags'

export default [
  httpDelete(string`/items/${state`currentItemId`}`)
  /*
                                PROPS: {
                                  response: {...}
                                }
                              */
]
```

````

**Expected behavior:**
Now everytime you run `prettier --write` the comment moves to the right.
Would expect that it stays in place.

**Comment by ikatyang** [comment]
Hi, thanks for the issue. This is a duplicate of #4080 and it's already fixed by #4083.

<details><summary>playground</summary>

**Prettier pr-4083**
[Playground link](https://deploy-preview-4083--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEBidACCAHOAnAQxgjwB0pyADagKwGdyBLAW2xJg2AwAsYZsAInAA2ceBgC+GAGZ4IzDAHIAAmHxwARoWEB6Xvx058REnUVNW7ThjowicSTLkLFavJu067AczPlycAAebHgcACZw0gQArsIcANrkGDx8giJicAAUtniMUN6UOozwzHQ6ACTAtvaUYNF47rAAkiVNYZQSlACUSRg6AFS9ycMjo2PjGAAKAEoA8pMAykicQxNr68nudGxQdHDLwAB0xxKrG+fDp1AXNxj9OuQAulTU5CAANCA4MIzQdMigAgNCAAd0mQIQ-xQBGEIIIAE9-p8CLZkFFhHtPloCGAANZiBbYHF5bzIGB4aJwT5hCBgNEwzEgPJ7UKTQjeZgEekYqkgeiBABChDxBIIzDgABk8nBuYyINF+AqAExkim8ol4FnIECcvC4mkgqAfEDYXKwADqjDCMG4yAAHAAGT6miB7c2EbDa01wFkANxln3cAEdoox3GyCByuUh0Yy9sxGLLeXQSaIAIrRCDwVWUz52DSW622pBKvOERjCEkAYXknO1UGgAZA0T2ABUCBoobG4BIJEA)
```sh
--arrow-parens always
--parser markdown
--no-semi
--single-quote
```

**Input:**
````markdown
### operator

```js
import { httpDelete } from '@cerebral/http/operators'
import { state } from 'cerebral/tags'

export default [
  httpDelete(string`/items/${state`currentItemId`}`)
  /*
                              PROPS: {
                                response: {...}
                              }
                            */
]
```

````

**Output:**
````markdown
### operator

```js
import { httpDelete } from '@cerebral/http/operators'
import { state } from 'cerebral/tags'

export default [
  httpDelete(string`/items/${state`currentItemId`}`)
  /*
                              PROPS: {
                                response: {...}
                              }
                            */
]
```

````

</details>

**Comment by ikatyang** [comment]
Duplicate of #4080
